### ISSUE TITLE
Improve Transfers page

### ISSUE THREAD

**Issue by amiskov** [cambiatus/frontend#431]
**Is your feature request related to a problem? Please describe.**
The "Transfer to a friend" page could be improved with some simple but convenient things.

**Describe the solution you'd like**
- [ ] Check the length of the memo (validate this field) before sending the data.
- [ ] Add a counter to the memo textarea (see creating an Action page as an example).
- [ ] While sending a transfer there will be a PIN popup. It has an error state "Enter 6 digits" before the actual entering those digits. This field doesn't have to be within an error state from the beginning. Also, we can make this field to be focused.
- [ ] Add frontend validation to all the fields and make sure it's consistent.

**Additional context**
Discovered while making #419.

**Comment by dev-danilosilva** [comment]
Some questions about this issue:

- Is there a limit to the number of characters on the memo field? If there is, what is a reasonable limit in your opinion ? ( Because since, it is an optional field, this is the only validation I can think of on that field)
-  The pin popup do not appears after the transfer confirmation. Is this a problem ?

**Comment by lucca65** [comment]
> Is there a limit to the number of characters on the memo field?

Yes. its 255 characters

> The pin popup do not appears after the transfer confirmation.  Is this a problem ?

**TLDR; No.** The PIN only appears if we need to grab the private key from the local storage, if we already have it no PIN will be requested. You can just reload the page after you sign up and the next transaction will always require the PIN

**Comment by dev-danilosilva** [comment]
I've added the character counter and a max length to the memo field. The users cannot submit an invalid or inconsistent transfer form data . I think that this is what this issue was about.

But, although the user cannot submit an invalid form, I think the next step that should be taken on this page is to show users what is wrong when they try send wrong data.

**Comment by lucca65** [comment]
@dev-danilosilva +1, I also see this issue as the right place to address error callbacks, which we don't support today. Its also not possible to try again, you have to reload the page. Can you fix this in your PR first? Of course if you feel comfortable doing so, let me know!

One good place you can draw inspiration from is the ActionEditor.elm file, there we show the error properly!

@amiskov any thoughts?

**Comment by dev-danilosilva** [comment]
@lucca65  For sure, I can fix it.
